doc/_templates/automodule.rst

```diff
@@ -5,24 +5,22 @@
    :no-members:
    :no-inherited-members:
 
-{% block functions %}
-{% if functions %}
+{% block classes %}
+{% if classes %}
 
 Classes
 -------
 
 .. autosummary:: 
    :template: autosummary.rst
    :toctree:
-
-{% for item in classes %}
-   {{ item }}
-{% endfor %}
+{% for item in classes %}{% if item not in ['zip', 'map', 'reduce'] %}
```

Comment:
for some reason they were popping up in the docs so I hard-coded them in...I'm not really sure why they were doing that